### DISCUSSION TITLE
Having problems passing the preprocessing for the student developer pack

### DISCUSSION THREAD

**Discussion by Mihajlo-Zivanovic** [community/community#56251] [GitHub Education]
### Hi everyone,

I've sent a picture that I created in MS Paint from an unedited picture of my student id*, together with a screenshot of a translation, since it is fully in cyrillic and wouldn't get registered without having it translated. However, it still denies it instantly, telling me to "complete my GitHub user profile with my full name and a short bio" and suggesting I use my camera (not an option, because my webcam is terrible). I have a very short bio and my account has my full name.
Any suggestions on what I might do to pass the preprocessing, since I don't really understand what is missing?

*where I live, we don't have id's but a booklet, like a passport, that are used for identification and proof of grades.

**Comment by samus-aran** [comment]
Hi @MihZivanovic 👋 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying. 👈 I suspect this is relevant. Let us know how you get on 👍
